utils/SwiftBuildSupport.py

```diff
@@ -187,7 +187,7 @@ def get_preset_options(substitutions, preset_file_names, preset_name):
         print_with_argv0("preset '" + preset_name + "' not found")
         sys.exit(1)
     if missing_opts:
-        print_with_argv0("missing option(s) for preset '" + preset_name + \
+        print_with_argv0("missing option(s) for preset '" + preset_name +
         "': " + ", ".join(missing_opts))
```

Comment:
Could you adjust the indentation on this continuation line while you are here?

This file LGTM.